aiokafka/protocol/message.py

```diff
@@ -233,10 +298,18 @@ def decode(cls, data, bytes_to_read=None):
         return items
 
     @classmethod
-    def repr(cls, messages):
+    def repr(
+        cls,
+        messages: Union[
+            io.BytesIO,
```

Comment:
Why `io.BytesIO` is here?

Reply:
Because `if isinstance(messages, io.BytesIO):` three lines below

Reply:
Looks like it's never used. It might be useful with `bytes` here (which is actually won't work). But "list of bytes stream" as input looks very strange to me. Could you imagine a case where we have a list of streams? Let's just drop this branch.

Reply:
Why "list of bytes stream", if it is just BytesIO or list of messages?

Reply:
`BytesIO` = stream